static/app/components/assigneeBadge.tsx

```diff
@@ -0,0 +1,136 @@
+// import {Fragment} from 'react';
+import {Fragment} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import ActorAvatar from 'sentry/components/avatar/actorAvatar';
+import Tag from 'sentry/components/badge/tag';
+import {Chevron} from 'sentry/components/chevron';
+import ExternalLink from 'sentry/components/links/externalLink';
+import Placeholder from 'sentry/components/placeholder';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Actor} from 'sentry/types';
+
+type AssigneeBadgeProps = {
+  chevronDirection: 'up' | 'down';
+  assignedTo?: Actor | undefined;
+  assignmentReason?: string;
+  showLabel?: boolean;
+};
+
+const AVATAR_SIZE = 16;
+
+export function AssigneeBadge({
+  assignedTo,
+  assignmentReason,
+  showLabel = false,
+  chevronDirection = 'down',
+}: AssigneeBadgeProps) {
+  const makeAssignedIcon = (actor: Actor) => {
+    return (
+      <Fragment>
+        <ActorAvatar
+          actor={actor}
+          className="avatar"
+          size={AVATAR_SIZE}
+          hasTooltip={false}
+        />
+        {showLabel && <Fragment>{actor.name}</Fragment>}
+        <Chevron direction={chevronDirection} size="small" />
+      </Fragment>
+    );
+  };
+
+  const makeUnassignedIcon = () => {
+    return (
+      <Fragment>
+        <Placeholder
+          shape="circle"
+          width={`${AVATAR_SIZE}px`}
+          height={`${AVATAR_SIZE}px`}
+        />
+        {showLabel && <Fragment>Unassigned</Fragment>}
+        <Chevron direction={chevronDirection} size="small" />
+      </Fragment>
+    );
+  };
+
+  const makeAssignedTooltipText = (actor: Actor) => {
+    // Cant use StyledTag's tooltipText prop because
+    // it screws with the nested div style ()
+    return (
+      <TooltipWrapper>
+        {tct('Assigned to [name]', {
+          name: actor.type === 'team' ? `#${actor.name}` : actor.name,
+        })}
```

Comment:
prefer t over tct if you're not using it to insert components
```suggestion
        {t('Assigned to %s', actor.type === 'team' ? `#${actor.name}` : actor.name)}
```

could even just do regular jsx
```suggestion
        {t('Assigned to')} {actor.type === 'team' ? `#${actor.name}` : actor.name}
```